content/en/docs/setup/additional-setup/gateway/index.md

```diff
@@ -0,0 +1,296 @@
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
+## Managing gateways
+
+The following describes how to manage gateways after installation. For more information on their usage, follow
+the [Ingress](/docs/tasks/traffic-management/ingress/) and [Egress](/docs/tasks/traffic-management/egress/) tasks.
+
+### Gateway selectors
+
+The labels chosen on the gateway pods are used by Gateway selectors, so it is important that your
+Gateway objects are configured to match these labels.
+
+For example, in the deployments above we set the `istio=ingressgateway` label on the pods.
+To apply a Gateway to these deployments, we would select the same label:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1beta1
+kind: Gateway
+metadata:
+  name: gateway
+spec:
+  selector:
+    istio: ingressgateway
+...
+{{< /text >}}
+
+### Gateway deployment topologies
+
+Depending on your mesh configuration and use cases, you may wish to deploy gateways in different ways.
+Below shows a few different patterns for gateway deployments.
+Note that multiple options can be utilized within the same cluster.
+
+#### Shared gateway
+
+In this model, a single centralized gateway is used by many applications, possibly across many namespaces.
+Gateway(s) in the `ingress` namespace delegate ownership of routes to application namespaces, but retain control over TLS configuration.
+
+{{< image width="50%" link="shared-gateway.svg" caption="Shared gateway" >}}
+
+This model works well when you have many applications you want to expose externally, as they are able to use shared infrastructure.
+It also works well in models that have the same domain or TLS certificates shared by many applications.
+
+#### Dedicated application gateway
+
+In this model, an application namespace has its own dedicated gateway installation.
+This allows giving full control and ownership to a single namespace.
+This level of isolation can be helpful for critical applications that have strict performance or security requirements.
+
+{{< image width="50%" link="user-gateway.svg" caption="Dedicated application gateway" >}}
+
+Unless there is another load balancer in front of Istio, this typically means that each application will have its own IP address,
+which may complicate DNS configurations.
+
+## Upgrading gateways
+
+### In place upgrade
+
+Because gateways utilize pod injection, new gateway pods that are created will automatically be injected with the latest configuration, which includes the version.
+
+To pick up changes to the gateway configuration, the pods can simply be restarted, using commands such as `kubectl rollout restart deployment`.
+
+If you would like to change the [control plane revision](/docs/setup/upgrade/canary/) in use by the gateway, you can set the `istio.io/rev` label on the gateway Deployment, which will also trigger a rolling restart.
+
+{{< image width="50%" link="inplace-upgrade.svg" caption="In place upgrade in progress" >}}
+
```

Comment:
We should highlight if users are changing gateway to be deployed from Istio-system to Istio-ingress, canary upgrade must be used.

Reply:
A service cannot select pods in another namespace so you cannot really do a cross namespace canary upgrade. If you want to do this you need a higher level control like
![2021-13-05_08-31-37](https://user-images.githubusercontent.com/623453/118148703-b0bc6d00-b3c5-11eb-8dc2-91ca900a930b.png)